### PR TITLE
Adding function and test function suscat

### PR DESCRIPTION
**Pull request by TBarasch**

### CODE REVIEW

File: pythcat/suscat.py [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
     """
     Detect suspected erroneous numeric data in user chosen columns of a dataframe
+
+     Parameters
+        ----------
+        df : Pandas dataframe object
+        col : list or array of column indices for which to test for suspected erroneous data
+        n : integer value for amount of suspected values to return
+        type : {'percent', 'number'}
+            This optional parameter specifies the whether n is a number of rows or percentage
```
Comment (EitharAlfatih) [inline pythcat/suscat.py]:
```suggestion
            This optional parameter specifies whether n is a number of rows or percentage
```

File: pythcat/suscat.py [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+
+    Returns
+    -------
+    dictionary with key as index of column and values as row indices of suspected erroneous values
+
+    Examples
+    --------
+    suscat(pd.DataFrame({'Age': [2, 23, 4, 11], 'Number': [11, 99, 23, 8]}), columns = [1], n = 2, type = 'percent')
+    > {1: [1,3]}
+    """
```
Comment (EitharAlfatih) [inline pythcat/suscat.py]:
```suggestion
>>>    suscat(pd.DataFrame({'Age': [2, 23, 4, 11], 'Number': [11, 99, 23, 8]}), columns = [1], n = 2, type = 'percent')
     {1: [1,3]}
    """
```

File: pythcat/suscat.py [hunk 3]
```diff
@@ -1,9 +1,71 @@
...
+    elif num == 'number':
+        alpha = (n+1)/((df.shape[0])+1)
+    for i in columns:
+        # isolate relevant column
+        a = df.iloc[:, i]
+        # find upper quantile value
+        h = np.quantile(a, 1-(alpha/2))
+        # find lower quantile value
+        l = np.quantile(a, alpha/2)
+        # extract indices through boolian comparison to the quantile values
```
Comment (EitharAlfatih) [inline pythcat/suscat.py]:
```suggestion
        # extract indices through boolean comparison to the quantile values
```

File: tests/test_suscat.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    """
+
+    df = pd.DataFrame({'x': [1, 2, 11, 22, 7], 'y': [100, 110, 120, 130, 140]})
+
+    out1 = suscat.suscat(df, [0, 1], n=80)
+    out2 = suscat.suscat(df, [0], num='number', n=2)
+
+
+    assert isinstance(out1, dict)
+    assert len(out1.keys()) == 2 and len(out1[1]) == 4
```
Comment (EitharAlfatih) [inline tests/test_suscat.py:38]: This line is failing for me

**Review by EitharAlfatih — CHANGES_REQUESTED** [review]

File: tests/test_suscat.py (hunk 4 above)
Comment (TBarasch) [reply tests/test_suscat.py:38]: testing again on my system, seems to pass ok

Comment (doraqmon) [reply tests/test_suscat.py:38]: pass on mine as well

**Review by EitharAlfatih — APPROVED** [review]
tests passed

**Pull request merged**